studio/schemas/objects/sections/contact-box.ts

```diff
@@ -0,0 +1,119 @@
+import { defineField } from "sanity";
+
+import { StringInputWithCharacterCount } from "studio/components/stringInputWithCharacterCount/StringInputWithCharacterCount";
+import { subtitleID, titleID } from "studio/schemas/fields/text";
+import { firstTranslation } from "studio/utils/i18n";
+
+export const contactBoxID = "contactBox";
+
+export enum ContactBoxBackground {
+  Dark = "dark",
+  Light = "light",
+}
+
+const backgroundOptions = [
+  { title: "Dark", value: ContactBoxBackground.Dark },
+  { title: "Light", value: ContactBoxBackground.Light },
+];
+
+export const contactBox = defineField({
+  name: contactBoxID,
+  title: "Contact Box",
+  type: "object",
+  fields: [
+    {
+      name: titleID.basic,
+      type: "internationalizedArrayString",
+      title: "Title",
+      description:
+        "Enter the primary title that will be displayed as large text in contact box.",
+    },
+    {
+      name: subtitleID.optional,
+      type: "internationalizedArrayString",
+      title: "Subtitle",
+      description: "Extra context under the main title.",
+    },
+    {
+      name: "background",
+      title: "Background",
+      type: "string",
+      description:
+        "Select the whether the contact box should have light or dark background.",
+      options: {
+        list: backgroundOptions,
+        layout: "radio",
+      },
+      initialValue: ContactBoxBackground.Dark,
+    },
+    {
+      name: "contactPoints",
+      title: "List of people to contact",
+      description: "List of tags and people to contact within those tags.",
+      type: "array",
+      of: [
+        {
+          type: "object",
+          fields: [
+            defineField({
+              name: "tag",
+              title: "Tag",
+              type: "string",
+              validation: (rule) => rule.max(30).required(),
+              components: {
+                input: (props) =>
+                  StringInputWithCharacterCount({ ...props, maxCount: 30 }),
+              },
+            }),
+            defineField({
+              name: "email",
+              title: "Enter the email address for contact point",
+              type: "string",
```

Comment:
You can use type: "email", and get validation right out of the box 😊

Reply:
Nice. Found it odd that I had to validate manually, but was just following a different scheme. But nice nice